dist/AureliaDependenciesPlugin.js

```diff
@@ -2,6 +2,7 @@
 Object.defineProperty(exports, "__esModule", { value: true });
 const IncludeDependency_1 = require("./IncludeDependency");
 const BasicEvaluatedExpression = require("webpack/lib/BasicEvaluatedExpression");
+const PreserveModuleNamePlugin_1 = require("./PreserveModuleNamePlugin");
```

Comment:
@pat841, please revert the changes in generated files, that are updated by release process